docs/user/ppl/admin/connectors/s3glue_connector.rst

```diff
@@ -14,10 +14,18 @@ S3Glue Connector
 Introduction
 ============
 
+Properties in DataSource Configuration
```

Comment:
This should be below introduction right. Can we move this to Glue Connector Properties in DataSource Configuration Section.